polling_stations/apps/data_importers/tests/test_run_new_imports.py

```diff
@@ -0,0 +1,134 @@
+import re
+from io import StringIO
+from unittest.mock import Mock, patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from data_importers.management.commands.run_new_imports import (
+    get_paths_changed,
+    git_rev_parse,
+    get_changed_scripts,
+    is_import_script,
+    any_import_scripts,
+    any_non_import_scripts,
+    Command as run_new_imports_command,
+)
+
+no_scripts = [
+    "polling_stations/apps/councils/management/commands/import_councils.py",
+    "polling_stations/apps/data_importers/base_importers.py",
+    "polling_stations/apps/data_importers/data_quality_report.py",
+    "polling_stations/apps/data_importers/management/commands/import_eoni.py",
+]
+all_scripts = [
+    "polling_stations/apps/data_importers/management/commands/import_barrow_in_furness.py",
+    "polling_stations/apps/data_importers/management/commands/import_wigan.py",
+]
+mix_of_scripts_and_not_scripts = [
+    "polling_stations/apps/data_importers/management/commands/import_barrow_in_furness.py",
+    "polling_stations/apps/data_importers/management/commands/import_eoni.py",
+    "polling_stations/apps/data_importers/management/commands/import_wigan.py",
+    "requirements/base.txt",
+]
+
+
+def test_get_paths_changed():
+    expected_empty = []
+    expected = [
+        "package-lock.json",
+        "polling_stations/apps/councils/management/commands/import_councils.py",
+        "polling_stations/apps/data_importers/base_importers.py",
+        "polling_stations/apps/data_importers/data_quality_report.py",
+        "polling_stations/apps/data_importers/management/commands/import_barrow_in_furness.py",
+        "polling_stations/apps/data_importers/management/commands/import_eoni.py",
+        "polling_stations/apps/data_importers/management/commands/import_wigan.py",
+        "polling_stations/settings/constants/councils.py",
+        "polling_stations/templates/base.html",
+        "polling_stations/templates/postcode_view.html",
+        "requirements/base.txt",
+    ]
+    assert expected_empty == get_paths_changed(
+        "1012c398b13d2e9c87f718b87e07ee9cd1c26222",
```

Comment:
I see what you've done here, but something feels…odd about this code checking it's own git history. I wonder what happens if, for whatever reason, we change the hashed (for example, what if we pull import scripts into a new repo? Maybe this PR is moot in that case, but you get my point).

To mitigate changes, can you leave a comment as to how you found these magic values?